qep-321-contributor-recognition.md

```diff
@@ -0,0 +1,80 @@
+# QGIS Enhancement 321: Contributor recognition and commit rights policies
+
+The QGIS project splits the concept of "notable developer recognition" from those with commit rights to the code
+repository. This is done in order to maintain good security practice for the code repository, with only those who
+are currently actively contributing to QGIS code having push/merge rights to the repository.
+
+Accordingly, there are two different types of QGIS developer status:
+
+## 1. QGIS Core Contributors
+
+"Core contributor" status is a lifetime status granted to developers who have demonstrated ongoing commitment and
+excellence in QGIS development.
+
+In order to be eligible, a developer must have:
+
+- 1.0.1. A history of high-quality code contributions to QGIS for a period of at least 12 months
+- 1.0.2. Demonstrated a high level of knowledge of c++, Python, the QGIS API, and QGIS development processes and practices
+- 1.0.3. Demonstrated a willingness to work in a friendly, collaborative manner and respond positively to change requests
+  and suggestions from other core contributors
+
+(While the project recognises the **immense** value in non-code contributions to QGIS, this status is designed exclusively for
+contributors of code.)
+
+## 1.1. Process for Nomination
+
+TODO
+
+# 2. Commit Rights
+
+This group of developers are those with write access to the main QGIS code repository. Unlike the "core contributor"
+status, this is a transient right and reflects only a developer's **current** activity within the QGIS project.
+**It is not a reflection of a developer's abilities, trustworthiness or skill, and is considered purely an
+administrative status designed to minimize security risk to the project while permitting responsive project
+maintenance.**
+
+- 2.0.1. Only developers who have previously been awarded the "core contributor" status are eligible for commit rights.
+- 2.0.2. When a new developer is initially awarded the "core contributor" status, they will also be granted commit rights
+to the code repository. However, QGIS developers may find other demands on their time or life circumstances and
+naturally reduce their involvement in the project. Over time, inactive accounts can represent a security risk,
+and maintaining accurate contact information becomes challenging. For these reasons, the following commit rights policy
+apply:
+
+## 2.1. Commit Rights Expiration
+
+- 2.1.1. Committers who have not made significant code contributions to the project **in the last 12 months** will have their
+commit rights automatically removed.
```

Comment:
We should probably detail a bit more what "significant code contributions" means, or at least give examples. Also, state on plural vs singular of "contribution", it has consequences.

"automatically removed" => change the wording, it is not automatic but a manual action from someone in the PSC.